instagram/src/components/Comments/CommentCotainer.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import "./Comments.css";
+import Comment from "./Comment";
+import CommentInput from "./CommentInput";
+
+class CommentContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: [],
+      commentText: "",
+      likes: 0
+    };
+  }
+
+  componentDidMount = () => {
+    this.setState({
+      comments: this.props.comments,
+      likes: this.props.likes
+    });
+  };
+
+  addNewComment = (e, index) => {
+    e.preventDefault();
+    const newComment = { text: this.state.commentText, username: "marcelo" };
+
+    this.setState(prevState => {
+      return {
+        active: false,
+        comments: [...prevState.comments, newComment],
+        commentText: ""
+      };
+    });
+  };
+
+  toggleClass() {
+    const currentState = this.state.active;
+    this.setState({ active: !currentState });
+  }
+
+  likePost = e => {
+    console.log("liked");
+    const currentState = this.state.active;
+
+    this.setState({
+      likes: this.state.likes + 1,
+      active: !currentState
```

Comment:
We can make use your `active` state to conditionally increment/decrement a like
```suggestion
      likePost = e => {
    console.log("liked");
    const currentState = this.state.active;

    !currentState
      ? this.setState({
          likes: this.state.likes + 1,
          active: !currentState
        })
      : this.setState({
          likes: this.state.likes - 1,
          active: !currentState
        });
  };
```